ui/component/formNewCollection/view.jsx

```diff
@@ -10,56 +10,65 @@ import { formatLbryUrlForWeb } from 'util/url';
 import Button from 'component/button';
 
 type Props = {
-  uri: string,
+  uri?: string,
+  sourceId?: string,
   onlyCreate?: boolean,
-  closeForm: (newCollectionName?: string) => void,
+  closeForm: (newCollectionName?: string, newCollectionId?: string) => void,
   // -- redux --
+  sourceCollectionName?: string,
   doPlaylistAddAndAllowPlaying: (params: {
-    uri: string,
+    uri?: string,
     collectionName: string,
     createNew: boolean,
     push: (uri: string) => void,
   }) => void,
 };
 
 function FormNewCollection(props: Props) {
-  const { uri, onlyCreate, closeForm, doPlaylistAddAndAllowPlaying } = props;
+  const { uri, sourceId, onlyCreate, closeForm, sourceCollectionName, doPlaylistAddAndAllowPlaying } = props;
 
   const {
     push,
     location: { search },
   } = useHistory();
 
   const buttonref: ElementRef<any> = React.useRef();
-  const newCollectionName = React.useRef('');
 
-  const [disabled, setDisabled] = React.useState(true);
+  const [newCollectionName, setCollectionName] = React.useState(
+    __('%copied_playlist_name% (copy)', { copied_playlist_name: sourceCollectionName }) || ''
+  );
```

Comment:
The fallback `|| ''` will never hit, since the front part will always be a valid string, right?

Reply:
Yep you are right